pom.xml

```diff
@@ -362,6 +362,21 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+
+		<!-- Spring boot actuator to expose metrics endpoint -->
+		<dependency>
```

Comment:
These dependencies are only needed in config-service/admin-service and portal, so they should be moved to `apollo-common`.

Besides, I think they also caused the travis-ci build failure for apollo-core module.